templates/typescript_gapic/package.json.njk

```diff
@@ -0,0 +1,40 @@
+{
+  "repository": "googleapis/nodejs-{{ api.naming.productName.toKebabCase() }}",
+  "name": "{{ api.naming.productName.toKebabCase() }}",
+  "version": "0.1.0",
+  "author": "Google LLC",
+  "description": "GAPIC {{ api.naming.productName }} client for Node.js",
```

Comment:
I don't . think `GAPIC` has any meaning to customers.  Maybe just `<product> . client . for . node.js?`